rednotebook/util/markup.py

```diff
@@ -146,7 +145,11 @@ def get_markup_for_day(day, with_text=True, with_tags=True, categories=None, dat
 
     # Add date if it is not None and not the empty string
     if date:
-        export_string += '= %s =\n\n' % date
+        # Following anchor placeholder will be used as a target for every entry reference
+        # mentioning this entry's date. In postprocessing it will be replaced by relevant HTML tag.
+        date_anchor = 'DATE_ANCHOR_PLACEHOLDER_%s' % day.date.strftime('%Y-%m-%d')
+
+        export_string += '= %s %s =\n\n' % (date_anchor, date)
```

Comment:
Shouldn't this be the other way around, i.e., date before date_anchor?

Reply:
Use new format string?

Reply:
In HTML export the date will be placed in `h1` tags, the anchor will be generated as invisible (empty) `span` tag - it will work anywhere near the date itself.
Formats done :)